packages/trip-details/src/index.tsx

```diff
@@ -330,6 +338,41 @@ export function TripDetails({
             }
           />
         )}
+        {co2 > 0 && co2Config.enabled && (
```

Comment:
You might want to throw a ? after co2config. Getting some white screen of death errors